lib/cn/cn_tree.c

```diff
@@ -1630,7 +1630,7 @@ static void
 cn_comp_update_split(
```

Comment:
Keep this one since it is a static function so this is probably the decl and the definition.